### DISCUSSION TITLE
插件 lantern:灯笼插件不动

### DISCUSSION THREAD

**Discussion by duchenpaul** [theme-shoka-x/hexo-theme-shokaX#26] [Q&A] [answered]
我的灯笼插件不会动 咋回事?
<img width="246" alt="image" src="https://user-images.githubusercontent.com/8120099/215755466-dfe8a2f9-33b5-46c8-ad1f-5fec63441e7b.png">

控制台输出:
<img width="1562" alt="image" src="https://user-images.githubusercontent.com/8120099/215755641-6ace9c98-0875-45f6-8cb1-a0eb5282371b.png">

**Comment by zkz098** [comment, marked as answer]
此问题已知，在上传到远程(例如gh pages等网页托管服务)后即可解决，可能为浏览器对localhost的gpu限制

**Reply by duchenpaul** [reply]
我这不是localhost 我是host in docker

**Reply by zkz098** [reply]
> 我这不是localhost 我是host in docker

没用过host in docker，但是我本地hexo s灯笼就是静止的，部署到远程就正常了
